.github/workflows/release.yml

```
@@ -1,77 +1,328 @@
+    linux-386
+    linux-amd64
+    linux-arm
+    linux-arm64
+    windows-386.exe
+    windows-amd64.exe
+    windows-arm64.exe
+
+concurrency:
+  group: release-${{ github.repository }}
+  cancel-in-progress: false
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)